core/src/main/java/com/novoda/noplayer/exoplayer/forwarder/BufferStateForwarder.java

```diff
@@ -0,0 +1,57 @@
+package com.novoda.noplayer.exoplayer.forwarder;
+
+import com.google.android.exoplayer2.ExoPlaybackException;
+import com.google.android.exoplayer2.ExoPlayer;
+import com.google.android.exoplayer2.PlaybackParameters;
+import com.google.android.exoplayer2.Timeline;
+import com.google.android.exoplayer2.source.TrackGroupArray;
+import com.google.android.exoplayer2.trackselection.TrackSelectionArray;
+import com.novoda.noplayer.listeners.BufferStateListeners;
+
+class BufferStateForwarder implements ExoPlayer.EventListener {
```

Comment:
Same here, not extending from the abstract `ExoPlayerEventForwarder` instead implementing the interface from `ExoPlayer` that we care about.